var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -76,9 +76,12 @@ def install(self, spec, prefix):
             'NVCC          = echo',
             'NVCCFLAGS     =',
         ])
-        if '+fpic' in spec:
+        if '+pic' in spec:
             make_args.extend(['CFLAGS=-fPIC', 'FFLAGS=-fPIC'])
```

Comment:
What is the right syntax to do it?? 
make_args.extend(['CFLAGS=???', 'FFLAGS=???'])